shell/browser/ui/cocoa/atom_ns_window_delegate.mm

```diff
@@ -138,6 +138,16 @@ - (void)windowDidResize:(NSNotification*)notification {
   shell_->NotifyWindowResize();
 }
 
+- (void)windowWillMove:(NSNotification*)notification {
+  NSWindow* window = [notification object];
+  NSSize size = [[window contentView] frame].size;
+  gfx::Rect new_bounds(gfx::Point(window.frame.origin), gfx::Size(size));
```

Comment:
@erickzhao don’t we need to use `gfx::ScreenRectFromNSRect` here as well?

Reply:
#19639

Reply:
I wasn't aware of that PR's changes! Will change things accordingly.